gruntfile.js

```diff
@@ -196,16 +198,4 @@ module.exports = function (grunt) {
   ])
 
   grunt.registerTask('default', ['generate-assets', 'concurrent:target'])
-
-  /**
-   * On watch, copy the asset that was changed, not all of them
-   */
```

Comment:
This did not seem to be doing its intended job. Instead it was totally preventing any changed assets to be updated during the watch (in my tests).

Asset copying now only deals with images, which one would expect to be added and edited infrequently, so it seems fine to just copy all assets rather than trying to be selective.